drivers/d3d12/rendering_device_driver_d3d12.cpp

```diff
@@ -5652,7 +5667,7 @@ void RenderingDeviceDriverD3D12::begin_segment(uint32_t p_frame_index, uint32_t
 	frames[frame_idx].desc_heap_walkers.aux.rewind();
 	frames[frame_idx].desc_heap_walkers.rtv.rewind();
 	frames[frame_idx].desc_heaps_exhausted_reported = {};
-	frames[frame_idx].null_rtv_handle = {};
+	frames[frame_idx].null_rtv_handle = { 0 };
```

Comment:
The decision to put `{}` was deliberate, My idea in cases like this is that you have to either find out what's the actual default value for a given type (which may not be apparent, is an impl detail, etc.) or just write the shortest thing than means, "I don't care which bytes end up there as long as they are deterministic." To me, `{0}` is kind of the latter, but not quite there. Or isn't this change due to stylistic reasons?

Reply:
`{}` will not work with MinGW.

Reply:
Oh, this naughty MinGW. Then we need a way for those regressions to stop happening. I can set a local MinGW toolchain up when I have time (as other poople contributing to D3D12, being the slipperiest, should do). Or maybe there's something doable on the CI front?

Reply:
```
error: ambiguous overload for 'operator=' (operand types are 'CD3DX12_CPU_DESCRIPTOR_HANDLE' and '<brace-enclosed initializer list>')
 5655 |         frames[frame_idx].null_rtv_handle = {};
```

Reply:
> Or maybe there's something doable on the CI front?

Currently, we only have MSVC CI setup, MinGW was removed to save CI build time, but it might be worth re-adding.

Reply:
I'm not enough of a language lawyer to really work out if the right one is MSVC or MinGW. Let's just hope we don't run into many of those cases.

PS: Wondering if `= CD3DX12_CPU_DESCRIPTOR_HANDLE{}` would have done the trick.

Reply:
> Wondering if = CD3DX12_CPU_DESCRIPTOR_HANDLE{} would have done the trick.

Yes, seems to be working. I'll update it a bit later.